platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/content/store/StoredFileFileSystemContentStore.java

```diff
@@ -27,34 +28,30 @@ public class StoredFileFileSystemContentStore implements IStoredFileContentStore
 	private final File directory;
 
 	/**
-	 * Constructs a new {@link StoredFileFileSystemContentStore}.
+	 * Constructs a new {@link StoredFileFileSystemContentStore} for the given
+	 * directory.
 	 *
-	 * @param directory
-	 *            an existing, writable directory (never <i>null</i>)
+	 * @param absolutePath
+	 *            the absolute path to an existing, writable directory (never
+	 *            <i>null</i>)
 	 */
-	public StoredFileFileSystemContentStore(File directory) {
+	StoredFileFileSystemContentStore(String absolutePath) {
 
-		this.directory = Objects.requireNonNull(directory);
-	}
-
-	@Override
-	public String getLocation() {
-
-		try {
-			return directory.getCanonicalPath();
-		} catch (IOException exception) {
-			throw new SofticarIOException(exception);
+		if (!absolutePath.startsWith("/")) {
```

Comment:
Hm, absoulte paths in windows always start with the drive, e.g. `C:\\`
Do we need to check OS here, or am I missing something?

Reply:
On Windows systems, we could use file URIs as exemplified here:
https://en.wikipedia.org/wiki/File_URI_scheme#Windows
I see no need to support a backslash based notation.